pkg/webui/console/components/device-data-form/index.js

```diff
@@ -348,6 +354,18 @@ class DeviceDataForm extends Component {
           name="supports_class_c"
           component={Checkbox}
         />
+        <Form.Field
```

Comment:
Do you think it makes sense to put these under another sub-heading like `Connections` or is it good as part of `General Settings`?

Reply:
Not really. IMO everything stored on the Identity Server is "General".

Also, it's not actually connections (that's only for the GS)